src/deserialize/parse.rs

```diff
@@ -262,15 +262,15 @@ fn read_bool_or_nil(
         .next()
         .ok_or_else(|| Error::ParseEdn("Could not identify symbol index".to_string()))?
         .0;
+    let delimiters = [',', ']', '}', ')', ';'];
     match c {
         't' if {
-            let val = chars.clone().take(4).map(|c| c.1).collect::<String>();
-            val.eq("rue ")
-                || val.eq("rue,")
-                || val.eq("rue]")
-                || val.eq("rue}")
-                || val.eq("rue)")
-                || val.eq("rue")
+            let val = chars
+                .clone()
+                .take_while(|(_, c)| !c.is_whitespace() && !delimiters.contains(c))
```

Comment:
I think the complete linter will ask you to replace this not AND not logic

Reply:
> 

I will rebase after #102 so we can let tooling audit.

Reply:
Actually, I don't see anything for 
cargo clippy -- -W clippy::all -W clippy::pedantic -W clippy::nursery --deny warnings
suggestions?